Exiled.API/Features/Map.cs

```diff
@@ -217,6 +217,25 @@ public static void TurnOffAllLights(float duration, IEnumerable<ZoneType> zoneTy
                 TurnOffAllLights(duration, zone);
         }
 
+        /// <summary>
+        /// Changes the <see cref="Color"/> of all lights in the facility.
+        /// </summary>
+        /// <param name="color">The new <see cref="Color"/> of the lights.</param>
+        public static void ChangeLightsColor(Color color)
+        {
+            foreach (RoomLightController light in RoomLightController.Instances)
+                light.NetworkOverrideColor = color;
+        }
+
+        /// <summary>
+        /// Resets the <see cref="Color">color</see> of all lights in the facility.
+        /// </summary>
+        public static void ResetLightsColor()
+        {
+            foreach (RoomLightController light in RoomLightController.Instances)
+                light.NetworkOverrideColor = Color.clear;
```

Comment:
I recall Color.clear is not the default color in game. Could you double-check it?

Reply:
actually, it is

Reply:
here's the Room.ResetColor()

        public void ResetColor()
        {
            Color = Color.clear;
        }

Reply:
My bad, that was another thing.